Automattic/vip-cli
# Fixes unit test error `bsdthread_register error` on Darwing x64

## Description

Unit tests were failing due to `bsdthread_register error`. Had to ~~recompile Darwin x64 binary from source: https://github.com/Automattic/go-search-replace~~ update binary from https://github.com/Automattic/go-search-replace/releases/

For more details about the error, see: https://github.com/golang/go/wiki/MacOS12BSDThreadRegisterIssue

## Steps to Test

1. Check out PR.
2. Run `npm test` on Darwin x64
3. Verify tests pass instead of giving `bsdthread_register` error
